guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb

```diff
@@ -0,0 +1,1555 @@
+{
```

Comment:
<p>The <code>area</code> property retrives the area of each buffer <strong>in the units of the DataFrame's spatial reference.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>